src/main/target/common_pre.h

```diff
@@ -204,7 +204,7 @@
 #define USE_OSD_HD
 #define USE_BLACKBOX
 
-#if TARGET_FLASH_SIZE > 512
```

Comment:
this is not the intention for this line.  Neither 448K or 512K targets should not have all this additional code included, the intention, IIRC, was for 1024K targets to include more by default.

as per discord converations, using `>= 1024` would more clearly state the intention.

Reply:
this can be seen by all the build failures.